src/Microsoft.AspNetCore.Authentication.OAuth/OAuthHandler.cs

```diff
@@ -63,16 +63,20 @@ public OAuthHandler(IOptionsMonitor<TOptions> options, ILoggerFactory logger, Ur
             var error = query["error"];
             if (!StringValues.IsNullOrEmpty(error))
             {
+                // TODO: Add first class properties for these?
```

Comment:
Yeah this would look a lot nicer if we added first class properties to AuthenticationProperties like the other ones there